R/calcIDataElecAndSteamGen.R

```diff
@@ -0,0 +1,56 @@
+#' calcIDataElecAndSteamGen
+#'
+#' Use data to derive OPENPROM input parameter iDataElecAndSteamGen
+#'
+#' @return  OPENPROM input data iDataElecAndSteamGen
+#'
+#' @author Anastasis Giannousakis, Fotis Sioutas, Giannis Tolios
+#'
+#' @examples
+#' \dontrun{
+#' a <- calcOutput(type = "IDataElecAndSteamGen", aggregate = FALSE)
+#' }
+#'
+#' @importFrom dplyr %>% select mutate row_number
+#' @importFrom tidyr crossing
+#' @importFrom quitte as.quitte
+#' @importFrom tibble tibble
+
+calcIDataElecAndSteamGen <- function() {
+
+  # filter years
+  fStartHorizon <- readEvalGlobal(system.file(file.path("extdata", "main.gms"), package = "mrprom"))["fStartHorizon"]
+  fStartY <- readEvalGlobal(system.file(file.path("extdata", "main.gms"), package = "mrprom"))["fStartY"]
+  
+  # Get country codes from mapping 
+  map <- toolGetMapping(name = "regionmappingOP.csv",
+                        type = "regional",
+                        where = "mappingfolder")
+  
+  
+  # Creating a dummy tibble with CHP capacities for each country
+  regions <- map$ISO3.Code
+  variable <- c("STE1AL", "STE1AH", "STE1AD", "STE1AR", "STE1AG", "STE1AB")
```

Comment:
these should be loaded from the `sets.gms` file found in the mrprom package files, hard-coding them can be a problem in the future